src/v2/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactNode } from "react"
+import { PropsWithChildren } from "react"
+import { PageBoundary } from "@/components/PageBoundary"
+// import { Header as NavHeader } from "@/components/Header"
+const NavHeader = () => <div className="h-16 border-border border-b">mocked header</div>
+
+/* -------------------------------------------------------------------------------------------------
+ * Root
+ * -----------------------------------------------------------------------------------------------*/
+
+const Root = ({ children }: PropsWithChildren) => (
+  <div className="flex min-h-screen flex-col">{children}</div>
+)
+
+/* -------------------------------------------------------------------------------------------------
+ * Header
+ * -----------------------------------------------------------------------------------------------*/
+
+const Header = ({ children }: PropsWithChildren) => (
```

Comment:
Let's call this `Hero` & don't include a `<header>` tag here, since we already use one inside our `Header` component